x-pack/legacy/plugins/uptime/server/rest_api/monitors/status.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { UMServerLibs } from '../../lib/lib';
+import { UMRestApiRouteFactory } from '../types';
+
+export const createGetMonitorRoute: UMRestApiRouteFactory = (libs: UMServerLibs) => ({
+  method: 'GET',
+  path: '/api/uptime/monitor/selected',
+  validate: {
+    query: schema.object({
+      monitorId: schema.maybe(schema.string()),
```

Comment:
We're specifying `monitorId` as `schema.maybe(...)`; I'm guessing ES will respond with an empty set but wasn't sure if you've tried that. It seems like it'd be ok to require this param, which is why I asked.